src/client/styles/components/Search.scss

```diff
@@ -6,12 +6,125 @@
 
   .nypl-text-field {
     input[type=text] {
-      height: 2rem;
       -webkit-box-shadow: none;
       box-shadow: none;
     }
   }
 
   .nypl-field-error {
   }
+
+  // Larger screen collapsed version of search form elements.
+
+  .nypl-omnisearch {
+    background-color: #fff;
+    margin-left: 0.25rem;
+  }
+
+  .nypl-omnisearch .nypl-omni-fields {
```

Comment:
You can nest CSS rules in sass and it's preferable. So you can do something like:

```
.nypl-omnisearch {
  .nypl-omni-fields {
    // ...

    input[type=text] {
      // ...
    }

    // ...
  }
  // ...
}
```